iotedgedev/modules.py

```diff
@@ -138,9 +138,17 @@ def build_push(self, no_build=False, no_push=False):
                 if not no_push:
                     # PUSH TO CONTAINER REGISTRY
                     self.output.info("PUSHING DOCKER IMAGE: " + tag)
+                    registry_key = None
+                    for key, registry in self.envvars.CONTAINER_REGISTRY_MAP.items():
+                        if registry.server.lower() == tag.split('/')[0].lower():
```

Comment:
Please add a sample of the string you are parsing here in a comment, so it's easier to understand the code.